src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java

```diff
@@ -27,7 +25,7 @@ public class ModuleClass implements Identity<ModuleClass>, Comparable<ModuleClas
 
     private final String className;
 
-    private final List<Session> sessions;
+    private final UniqueList<Session> sessions;
```

Comment:
For consistency, should this be changed as follows?
```suggestion
    private final UniqueList<Session> sessions = new UniqueList<>();
```